### PR TITLE
[ADD] Expenses: reimburse employees doc

### PR DESCRIPTION
**Pull request by larm-odoo**
Original Expenses doc too long, breaking off the reimbursing expenses  to accounting section into its own doc, and making necessary updates to version 17 (original was written for 16). Also updating per new doc guidelines.

[Project task](https://www.odoo.com/odoo/project/3835/tasks/4078251?cids=3) card for this PR.

### CODE REVIEW

**Comment by larm-odoo** [comment]
Hi @hojo-odoo - this is ready for a first-round review!

File: content/applications/finance/expenses/reimburse.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reimburse.rst:6]:
```suggestion
to reimburse the employee. Just like approving and posting expenses, employees can be reimbursed in two ways:
```

File: content/applications/finance/expenses/reimburse.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reimburse.rst:8]:
```suggestion
either with cash, check, or direct deposit (:ref:`individually <expenses/reimburse-single>` or
:ref:`in bulk <expenses/reimburse-bulk>`), or :ref:`reimbursed in a payslip
```

File: content/applications/finance/expenses/reimburse.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reimburse.rst:48]:
```suggestion
  <employees/private-info>` of the **Employees** app, that bank account populates this field, by
```

File: content/applications/finance/expenses/reimburse.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reimburse.rst:80]:
```suggestion
the number of selected expense reports appears on a :guilabel:`(#) Selected` button in the
```

File: content/applications/finance/expenses/reimburse.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reimburse.rst:91]:
```suggestion
- :guilabel:`Journal`: Select the accounting journal to post the payment to, using the drop-down
```

File: content/applications/finance/expenses/reimburse.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reimburse.rst:130]:
```suggestion
:guilabel:`Posted` (and then :guilabel:`Done`), until the paycheck is processed.
```

**Review by hojo-odoo — APPROVED** [review]
Hey @larm-odoo,

Nice job on this PR. I'm approving with a few small suggestions.

Thanks,
John

File: content/applications/finance/expenses/reimburse.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reimburse.rst:76]:
```suggestion
   Adjusting the :guilabel:`STATUS` filters to only show :guilabel:`Posted` expense reports is not
```

File: content/applications/finance/expenses/reimburse.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reimburse.rst:79]:
```suggestion
Tick the checkbox next to the :guilabel:`Employee` column title to select all the reports in the list. Once ticked,
```

File: content/applications/finance/expenses/reimburse.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reimburse.rst:82]:
```suggestion
the number of selected expense reports appears at the top of the page (:guilabel:`(#) Selected`).
Additionally, a :guilabel:`Register Payment` button also appears in the upper-left corner.
```

File: content/applications/finance/expenses/reimburse.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reimburse.rst:91]:
```suggestion
- :guilabel:`Journal`: Select the accounting journal the payment should be posted to, using the drop-down
```

File: content/applications/finance/expenses/reimburse.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reimburse.rst:94]: Again, I'm also seeing SEPA Credit Transfer as an option here -- but, perhaps, that's only for BE-based businesses. Still, something to look into during your revisions.

File: content/applications/finance/expenses/reimburse.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reimburse.rst:95]:
```suggestion
- :guilabel:`Group Payments`: When multiple expense reports are selected for the same employee, this
```

File: content/applications/finance/expenses/reimburse.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reimburse.rst:96]:
```suggestion
  option appears. Tick the checkbox to have only one payment made, rather than
```

File: content/applications/finance/expenses/reimburse.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reimburse.rst:99]:
```suggestion
  field, by default.
```

File: content/applications/finance/expenses/reimburse.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reimburse.rst:105]:
```suggestion
When the fields on the pop-up window are completed, click the :guilabel:`Create Payments` button to register the payments,
```

File: content/applications/finance/expenses/reimburse.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reimburse.rst:114]:
```suggestion
If the *Reimburse in Payslip* option is activated on the *Settings* page, payments can be added to their next payslip, instead of issued manually.
```

File: content/applications/finance/expenses/reimburse.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reimburse.rst:118]:
```suggestion
   status of *Approved*. Once an expense report has a status of *Posted*, the
```

File: content/applications/finance/expenses/reimburse.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reimburse.rst:121]:
```suggestion
Navigate to :menuselection:`Expenses app --> Expense Reports`, and click on the
```

File: content/applications/finance/expenses/reimburse.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reimburse.rst:123]:
```suggestion
Next Payslip` button, and the expenses are added to the next payslip issued for that employee.
```

Comment (ksc-odoo) [reply content/applications/finance/expenses/reimburse.rst:123]: Do you think it would be beneficial to add that a comment about it being added to their next payslip appears in teh chatter?

File: content/applications/finance/expenses/reimburse.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reimburse.rst:130]:
```suggestion
The status for the expense report remains :guilabel:`Approved`. The status only changes to
:guilabel:`Posted` (and then :guilabel:`Done`) when the paycheck is processed.
```

File: content/applications/finance/expenses/reimburse.rst
Comment (larm-odoo) [reply content/applications/finance/expenses/reimburse.rst:32]: I feel like I am going crazy, because I definitely saw that in one of the DB's I was working with! I just rechecked the runbots for today, and my own test DB, and you are correct- I honestly don't know where that came from, except I know I definitely saw it. I feel like I said I saw a ghost =D

File: content/applications/finance/expenses/reimburse.rst
Comment (larm-odoo) [reply content/applications/finance/expenses/reimburse.rst:45]: I rechecked with my blank database, and only manual and check are the default options. I actually had to remove NACHA because that's from the Belgian localizations in all our Runbots.

**Comment by larm-odoo** [comment]
Finally, all cleaned up and conflicts fixed for a final review, @samueljlieber!

File: content/applications/finance/expenses/reimburse/pay-via-payslip.png
Comment (samueljlieber) [inline content/applications/finance/expenses/reimburse/pay-via-payslip.png:1]: This image needs to be compressed (`reimburse/pay-via-payslip.png`)

File: content/applications/finance/expenses/reimburse/payment.png
Comment (samueljlieber) [inline content/applications/finance/expenses/reimburse/payment.png:1]: This image is too wide at 1447px, please reduce to be at or under 933px (`reimburse/payment.png`)

**Review by samueljlieber — APPROVED** [review]
Hi @larm-odoo, nice job on this Expenses PR to break out the Reimburse content to its own doc.

Approving with two quick mandatory image fixes, please implement these before merge, thanks!
.....
@robodoo delegate=larm-odoo

**Comment by larm-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
